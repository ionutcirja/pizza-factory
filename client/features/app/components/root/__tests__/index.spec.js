import React from 'react';
import { shallow } from 'enzyme';
import App from '..';

describe('App component', () => {
  const Root = () => (<span>root component</span>);
  const Page = () => (<span>page component</span>);
  
  const propsToRender = {
    routes: [
      {
        key: 'root',
        path: '/',
        component: Root,
      },
      {
        key: 'page',
        path: '/page',
        component: Page,
      },
    ],
  };
  
  it('should render a collection of routes', () => {
    const wrapper = shallow(<App {...propsToRender} />);
    expect(wrapper).toMatchSnapshot();
  });
});
