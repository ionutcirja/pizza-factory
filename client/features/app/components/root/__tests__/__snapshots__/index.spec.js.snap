// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render a collection of routes 1`] = `
<Switch
  key="routes"
>
  <Route
    component={[Function]}
    key="root"
    path="/"
  />
  <Route
    component={[Function]}
    key="page"
    path="/page"
  />
</Switch>
`;
